Gameplay/misc/GauntletManager.cpp

```diff
@@ -130,6 +133,7 @@ void Hachiko::Scripting::GauntletManager::CheckRoundStatus()
 	if (current_round >= _enemy_packs.size()) {
 		completed = true;
 		OpenDoors();
+		if (_closing_door) _closing_door->SetActive(false);
```

Comment:
```suggestion
		if (_closing_door)
		{
			_closing_door->SetActive(false);
			_closing-door->ChangeDissolveProgress(1.0f);
		}
```

Don't know if this works but it will be great that it dissolves like the door one >.<
Thank you and sorry

Reply:
Or change laser state to Dissolving it will work better as @VBoschMatas programmed it